src/components/Settings.css

```diff
@@ -38,3 +38,12 @@
 .Settings-input-error {
   border-color: #cc0000;
 }
+
+.Settings-dropdown {
```

Comment:
Similar to the `.Settings-input`, you could add `outline: none !important;` to prevent the dropdown showing an outline on focus